i18n-helpers/src/lib.rs

```diff
@@ -148,13 +148,33 @@ pub enum Group<'a> {
     ///
     /// This includes `[Text("foo")]` as well as sequences with text
     /// such as `[Start(Emphasis), Text("foo") End(Emphasis)]`.
-    Translate(Vec<(usize, Event<'a>)>),
+    Translate(Vec<(usize, Event<'a>)>, GroupExtra),
```

Comment:
Since both enum variants have the same fields, but only one of them seems to need the `GroupExtra` data, I think we could refactor `Group` into a struct in the future.

Something like

```rust
enum Kind { Translate, Skip }

struct Extra {
   pub translator_comments: Vec<String>,
}

struct Group {
    kind: Kind,
    extra: Extra,
}
```

That should give less repetition in the code that uses these values.

You could perhaps just skip the `GroupExtra` field on the `Skip` variant?

Reply:
Let me drop the `GroupExtra` from `Skip` for now; at the moment, we do not need it.  I overgeneralized there.